patch/files/scripts/outside.gd

```diff
@@ -2653,6 +2711,12 @@ func _on_details_pressed(empty = null):
 		get_node("playergroupdetails/quicksell").set_disabled(false)
 	else:
 		get_node("playergroupdetails/quicksell").set_disabled(true)
+	#ralph4
+	if get_parent().get_node("explorationnode").currentzone.code in ['wimborn','gorn','frostford'] && globals.state.capturedgroup.size() > 0:

```

Comment:
We should probably add a condition to the start of this chain of ifs that checks if useRalphsTweaks is false, and if so it disables the button and adds a tooltip explaining that it is specific to ralph's.

Reply:
Fixed, added a to-do to add a tooltip, that'll be a gui change requiring the godot engine to edit right?

Reply:
No, get_node("playergroupdetails/bountysell").set_tooltip("This feature requires enabling Ralph's Tweaks.")

Reply:
You fixed this one